content/download/weekly/index.adoc

```diff
@@ -0,0 +1,50 @@
+---
+layout: simplepage
+title: Weekly Release Line
+---
+:toc:
+
+The Jenkins _Weekly Release Line_ delivers bug fixes and new features rapidly to users and plugin developers who need them.
+These releases are generally delivered on a weekly cadence, hence the name.
+For more conservative users, there is also a 
+link:/download/lts/[Long-Term Support (LTS)] line.
+
+## Model
+
+The "weekly" release cadence is **aspirational**.
+Generally we target releasing Jenkins every week, but sometimes we do out-of-order releases.
+To get notifications about new releases, it is possible to subscribe to link:https://www.jenkins.io/changelog/rss.xml[releases RSS feed].
+
+Conditions for new releases of the _Weekly Release Line_:
+
+* **Regular releases** with all features, improvements and bug fixes integrated since the last release.
+** Currently these releases happen on a weekly basis on Tuesdays.
+* **Security releases** for the Jenkins core (see the link:/security[Jenkins Security] page).
+** We coordinate Jenkins Weekly and LTS releases so that the vulnerabilities are fixed at the same time in all release lines.
+   Jenkins security team sends notifications about upcoming security releases to link:/security/#advisories[Jenkins Security Advisory Channels].
+** Usually these releases happen on Wednesdays.
+   If we release security updates on Wednesday, there will be no regular release the day before.
+* **Out-of-order releases** in the case of major regressions and/or other reasons.
+
+Weekly releases are managed by Jenkins core maintainers
+who review the suggested pull requests, integrate changes and deliver the releases.
+The team also manages link:/changelog[changelogs] and the LTS backporting process.
+See link:https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc[this page] for more information about Jenkins core maintenance.
+
+## No Backporting
+
+Unlike for the LTS line,
+there is NO backporting process for the _Weekly Release Line_.
```

Comment:
```suggestion
there is no backporting process for the _Weekly Release Line_.
```

IMO allcaps is weird on the web when you could use formatting to emphasize. (I also disagree that this needs special emphasize given it has its own section header that says basically the same thing, so I left it out).